packages/calcite-components/src/components/pick-list-item/pick-list-item.scss

```diff
@@ -1,3 +1,15 @@
+/**
+ * CSS Custom Properties
```

Comment:
I thought we weren't doing pick-list because its deprecated?
What about the list-item though? it has a calcite-action inside of it. Did you search for any component using calcite-action?

Reply:
I forgot we weren't doing picklist (I've now updated Monday to put all pic list elements on hold). Yes, I determined component dependencies by searching `calcite-` in every component and making a list of the other calicte-components in it's shadow dom.

Reply:
Hey @alisonailea when I search I come up with `calcite-list-item` containing `calcite-action`.

Reply:
The dependency graphs show up in the component `readme.md` files: https://github.com/Esri/calcite-design-system/tree/main/packages/calcite-components/src/components/list-item#graph